Libraries/Components/Touchable/Touchable.js

```diff
@@ -331,7 +331,7 @@ var TouchableMixin = {
         } else if (evt.eventType === 'blur') {
           cmp.touchableHandleActivePressOut && cmp.touchableHandleActivePressOut(evt);
         } else if (evt.eventType === 'select') {
-          cmp.touchableHandlePress && cmp.touchableHandlePress(evt);
+          cmp.touchableHandlePress && !cmp.props.disabled && cmp.touchableHandlePress(evt);
```

Comment:
@kmagiera Note: I'm not exactly sure this is the correct way to handle that, as it relies on `cmp.props` directly

Reply:
Maybe call to this `touchableHandleStartShouldSetResponder()` instead?

Reply:
or nevermind, this looks ok to me, TouchableMixin is already accesing that prop in this method: `touchableHandleStartShouldSetResponder`